predicators/settings.py

```diff
@@ -328,6 +328,12 @@ class GlobalSettings:
     sticky_table_place_sticky_fall_prob = 0.05
     sticky_table_pick_success_prob = 0.9
     sticky_table_tricky_floor_place_sticky_fall_prob = 0.5
+    sticky_table_num_tables = 5  # cannot be less than 3
```

Comment:
I think there was a merge conflict at some point because some of these are redundant with the lines above